homeassistant/components/overkiz/sensor.py

```diff
@@ -413,6 +413,13 @@ class OverkizSensorDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
     ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_REMAINING_HOT_WATER,
+        name="Warm Water Remaining",
```

Comment:
Use capitalisation consistently with the other sensors
```suggestion
        name="Warm water remaining",
```

Reply:
Yeah, sorry, my bad. Fixed!